addons/payment/views/payment_provider_views.xml

```diff
@@ -48,11 +48,11 @@
                             </div>
                         </button>
                     </div>
-                    <field name="image_128" widget="image" class="oe_avatar"/>
-                    <widget name="web_ribbon" title="Disabled" bg_color="bg-danger" attrs="{'invisible': [('state', '!=', 'disabled')]}"/>
-                    <widget name="web_ribbon" title="Test Mode" bg_color="bg-warning" attrs="{'invisible': [('state', '!=', 'test')]}"/>
+                    <field name="image_128" widget="image" class="oe_avatar" attrs="{'readonly': [('module_state', '=', 'uninstalled')]}"/>
```

Comment:
Does this change anything? I tested on a runbot without your patch, and I can't update the image of an uninstalled provider even if it is _not_ readonly.

Reply:
currently the image is readonly as the banner is on top of it, once i apply the invisible attribute for the ribbon to display only when the module is installed, the image is getting editable even if module is not installed.

Reply:
thanks for the feedback @AntoineVDV , as suggested i will update the PR and will let you know